snmp/assets/dashboards/interface_performance.json

```diff
@@ -0,0 +1,521 @@
+{
+  "title": "Interface Performance",
```

Comment:
Minor nit, most dashboards have the integration name in it's title. Would it be worth calling this `SNMP Interface Performance`?

Reply:
@naaltman should we change it for this and for datacenter ?

Reply:
I don't think we want to include SNMP here -- while SNMP is the source of the data we collect, this is part of the larger Network Device Monitoring product, and SNMP is really an implementation detail in that sense.